Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -0,0 +1,175 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class JumpBackInTests: BaseTestCase {
+    func closeKeyboard() {
+        waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
+        navigator.performAction(Action.CloseURLBarOpen)
+        navigator.nowAt(NewTabScreen)
+    }
+
+    override func setUp() {
+        super.setUp()
+
+        closeKeyboard()
+
+        // Since I can't scroll on M1, closing the "Switch Your Default Browser"
+        // box makes the "Jump Back In" section visible without scrolling in the
+        // last part of the test.
+        if app.staticTexts["Switch Your Default Browser"].exists {
+            app.buttons["Close"].firstMatch.tap()
+        }
+
+        // "Jump Back In" is enabled by default. See Settings -> Homepage
+    }
+
+    func testJumpBackInSection() {
+        // Open a tab and visit a page
+        navigator.openURL("https://www.example.com")
+        waitUntilPageLoad()
+
+        // Open a new tab
+        navigator.goto(TabTray)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
+        closeKeyboard()
+
+        // "Jump Back In" section is displayed
+        waitForExistence(app.staticTexts["Jump Back In"])
+
+        // The contextual hint box is not displayed consistently, so
+        // I don't test for its existence.
```

Comment:
The test on TestRail requires us to check the contextual hint, but the contextual hint does not appear 100% of the time.